src/libtls/tests/suites/test_hkdf.c

```diff
@@ -0,0 +1,429 @@
+/*
+ * Copyright (C) 2020 Pascal Knecht
+ * Copyright (C) 2020 Méline Sieber
+ *
```

Comment:
```suggestion
 * HSR Hochschule fuer Technik Rapperswil
 *
```
Also in the other copyright headers.